### ISSUE TITLE
Improve docs: a11y: toggle label on default code example background does not meet color contrast

### ISSUE THREAD

**Issue by jbadan** [SAP/fundamental-react#461]
### Description
Background toggle has insufficient color contrast:  4.48 (foreground color: #6a6d70, background color: #edeef0, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1

We should change the default background color for code examples.

<img width="1154" alt="screen shot 2019-03-06 at 9 39 01 am" src="https://user-images.githubusercontent.com/29607818/53897594-b521ba00-3ff3-11e9-9dba-45ba022a4b89.png">

### Versions
**fundamental-react:** 0.4.0
**fiori-fundamentals:** N/A

---

_**NOTE:** Where applicable, please include uncropped screen captures._

_**DISCLAIMER:**
After triaging an issue, the fundamental-react team will see if it can be reproduced or confirmed. If more information is needed, the fundamental-react team will contact the author.  Any issues awaiting responses from the author for more than 7 days will be closed.  The author can re-open the issue at a later time if they can present the requested information._
